_data/trans/de.json

```
@@ -22,10 +22,10 @@
     "guideexplanations": "Die Links oben sind farbkodiert, um die Schwierigkeit des Account-Löschens anzuzeigen:",
     "guidehard": "Kann nicht gelöscht werden, ohne den jeweiligen Support zu kontaktieren",
     "guideimpossible": "Kann nicht gelöscht werden",
-    "guidelimited": "Account can only be deleted if you live in a location with privacy rights or are covered by an applicable policy such as the iOS App Store guidelines",
+    "guidelimited": "Account kann nur gelöscht werden, wenn du an einem Ort mit Datenschutzrechten lebst oder von zutreffenden Richtlinien, wie z.B. den iOS App Store Richtlinien, abgedeckt wirst.
```

```suggestion
    "guidelimited": "Account kann nur gelöscht werden, wenn du an einem Ort mit Datenschutzrechten lebst oder von zutreffenden Richtlinien, wie z.B. den iOS App Store Richtlinien, abgedeckt wirst.",
```

Missing end quote and comma.